finality-aleph/src/sync/mock/mod.rs

```diff
@@ -4,7 +4,7 @@ use codec::{Decode, Encode};
 use sp_core::H256;
 
 use crate::{
-    sync::{ChainStatusNotification, Header, Justification as JustificationT},
+    sync::{Block as BlockT, ChainStatusNotification, Header, Justification as JustificationT},
```

Comment:
Aaaa, `Header` but `JustificationT`, there was already an inconsistency here and you are only adding to it, but please, make this consistent. >.<

I soft prefer the version without `T`, since it's not necessary, but any consistent approach is acceptable.

Reply:
fixed